name: Supervised Classification, features: eucalyptus
{'X_train': {'cols': 0.02962595167163191, 'rows': 0.12421383647798742}, 'X_valid': {'cols': 0.022840119165839123, 'rows': 0.11320754716981132}, 'X_test': {'cols': 0.03175057208237986, 'rows': 0.1358695652173913}}
dataset sizes
(645, 19) (159, 19) (184, 19)
dropped dataset sizes
(566, 19) (141, 19) (159, 19)
key: 8925, k: 1/4, dataset: 188, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 9, 'max_bin': 169, 'max_depth': 29, 'min_data_in_leaf': 14, 'learning_rate': 0.001628247228100923, 'num_iterations': 358, 'objective': 'softmax', 'verbose': -1, 'num_class': 5}
Training until validation scores don't improve for 100 rounds
Did not meet early stopping. Best iteration is:
[358]	valid_0's multi_logloss: 1.03317
strategy:None, acc gbm: 0.6195652173913043
strategy:None, nll xbg:1.1740853786468506
name: Supervised Classification, features: eucalyptus
{'X_train': {'cols': 0.025743707093821514, 'rows': 0.11024844720496894}, 'X_valid': {'cols': 0.022883295194508012, 'rows': 0.11801242236024845}, 'X_test': {'cols': 0.028318077803203664, 'rows': 0.11413043478260869}}
dataset sizes
(665, 19) (161, 19) (184, 19)
dropped dataset sizes
(593, 19) (142, 19) (163, 19)
key: 6176, k: 2/4, dataset: 188, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 9, 'max_bin': 169, 'max_depth': 29, 'min_data_in_leaf': 14, 'learning_rate': 0.001628247228100923, 'num_iterations': 358, 'objective': 'softmax', 'verbose': -1, 'num_class': 5}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Did not meet early stopping. Best iteration is:
[358]	valid_0's multi_logloss: 1.00319
strategy:None, acc gbm: 0.5543478260869565
strategy:None, nll xbg:1.2056909799575806
name: Supervised Classification, features: eucalyptus
{'X_train': {'cols': 0.026648900732844767, 'rows': 0.10759493670886076}, 'X_valid': {'cols': 0.026982011992005332, 'rows': 0.12658227848101267}, 'X_test': {'cols': 0.02974828375286041, 'rows': 0.11956521739130435}}
dataset sizes
(645, 19) (158, 19) (184, 19)
dropped dataset sizes
(576, 19) (138, 19) (162, 19)
key: 4386, k: 3/4, dataset: 188, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 9, 'max_bin': 169, 'max_depth': 29, 'min_data_in_leaf': 14, 'learning_rate': 0.001628247228100923, 'num_iterations': 358, 'objective': 'softmax', 'verbose': -1, 'num_class': 5}
Training until validation scores don't improve for 100 rounds
Did not meet early stopping. Best iteration is:
[358]	valid_0's multi_logloss: 1.02839
strategy:None, acc gbm: 0.532608695652174
strategy:None, nll xbg:1.230624794960022
name: Supervised Classification, features: eucalyptus
{'X_train': {'cols': 0.02246469833119384, 'rows': 0.10213414634146341}, 'X_valid': {'cols': 0.0282413350449294, 'rows': 0.10975609756097561}, 'X_test': {'cols': 0.03832951945080091, 'rows': 0.14673913043478262}}
dataset sizes
(685, 19) (164, 19) (184, 19)
dropped dataset sizes
(616, 19) (146, 19) (157, 19)
key: 9581, k: 4/4, dataset: 188, missing: None, impute: None
training gbm for 5000 epochs
{'num_leaves': 9, 'max_bin': 169, 'max_depth': 29, 'min_data_in_leaf': 14, 'learning_rate': 0.001628247228100923, 'num_iterations': 358, 'objective': 'softmax', 'verbose': -1, 'num_class': 5}
Training until validation scores don't improve for 100 rounds
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
/home/jahan/.conda/envs/jax/lib/python3.10/site-packages/lightgbm/engine.py:177: UserWarning: Found `num_iterations` in params. Will use it instead of argument
  _log_warning(f"Found `{alias}` in params. Will use it instead of argument")
Did not meet early stopping. Best iteration is:
[358]	valid_0's multi_logloss: 1.02034
strategy:None, acc gbm: 0.6902173913043478
strategy:None, nll xbg:1.120044469833374